src/generate/renders/generated/CliCustom.ts

```diff
@@ -148,17 +149,19 @@ function ConstructMethodBodyParameter(model: CodeModelAz, needGeneric = false, r
                     originalParameterStack.pop();
                     originalParameterNameStack.pop();
                 }
-                // if (!addGenericSchema && needGeneric && !isNullOrUndefined(model.CommandGroup.language['az']['genericTargetSchema']) && model.Method.extensions?.['cli-split-operation-original-operation']?.['genericSetterParam'] !== model.MethodParameter) {
-                //     originalParameterStack.push(
-                //         new Parameter(model.CommandGroup.language['az']['genericTargetSchema'].language['python']['name'],
-                //         model.CommandGroup.language['az']['genericTargetSchema'].language['python']['description'],
-                //         model.CommandGroup.language['az']['genericTargetSchema']
-                //     ));
-                //     originalParameterNameStack.push(model.CommandGroup.language['az']['genericTargetSchema'].language['python']['name']);
-                //     addGenericSchema = true;
-                // }
-                originalParameterStack.push(model.MethodParameter);
-                originalParameterNameStack.push(model.MethodParameter_Name);
+                if (
+                    originalParameterStack.length === 0 &&
+                    allParams.indexOf(model.MethodParameter.schema) === -1
+                ) {
+                    allParams.push(model.MethodParameter.schema);
+                    originalParameterStack.push(model.MethodParameter);
+                    originalParameterNameStack.push(model.MethodParameter_Name);
+                } else if (originalParameterStack.length > 0) {
```

Comment:
not quite understand this logic, why check stack length >0 here?

Reply:
the originalParameterStack is used to store those unbuild parameters' original flattened parameter.

Let's say the object is like the following obj1, and all the properties of obj1 has been flattened to the top level. 
when we handle parameter B, originalParameterStack will have one element A,
when we handle parameter C, originalParameterStack will have two elements A and B
when we handle parameter D, we will found B has finished construction, we will pop B out of originalParameterStack, only A left again. 

originalParameterStack.length > 0 means this current process parameter is not the original top level body parameter, and we only want to reduce the same parameter construction. 
let's say we have two operation `op1` and `op2`, they are about to be merged into the same cli command.
op1 body parameter's schema is a complex object obj1
```
{
   A: {
       B: {
           C
       },
       D
   }
}
```
op2 body parameter 
```
{
   E: {
       A: {
           B: {
               C
           },
           D
       }
   }
}
```
If we don't add originalParameterStack.length > 0 condition, we will find that A has already been record in allParams, then we will not add A, which will cause the construction of E is incorrect.